manage_app.py

```diff
@@ -1,33 +1,61 @@
 #!/usr/bin/env python3
-import subprocess
-import requests
-import time
+# mypy: ignore-errors
 import os
 import smtplib as smtp
 import ssl
-import os
+import subprocess
+import threading
+import time
+import typing
+from argparse import ArgumentParser
 from email.mime.multipart import MIMEMultipart
 from email.mime.text import MIMEText
-import typing
-import threading
+from logging import getLogger
+
+import requests
 import schedule
+from dotenv import load_dotenv
 
 BUILD_RUNNING = threading.Event()
 CURRENTLY_BUILDING = threading.Event()
 BUILD_FAILED = threading.Event()
 lock = threading.Lock()
 BUILD_TRY = 0
+logger = getLogger("manage-app-logger")
```

Comment:
Where are you using this logger? If you're going to be using a logger, swap all print statements for the logger. Otherwise, remove the logger. I prefer the latter.